app/boards/intel_adsp_cavs25.conf

```diff
@@ -30,6 +30,7 @@ CONFIG_COMP_VOLUME_WINDOWS_FADE=y
 CONFIG_COMP_DRC=y
 CONFIG_COMP_CROSSOVER=y
 CONFIG_COMP_MULTIBAND_DRC=y
+CONFIG_COMP_ARIA=y
```

Comment:
I think this needs to be a separate git comit at least. Enabling Aria on tgl/tgl-h is not related to the main commit.

Reply:
Yep, it's moved to 2nd commit now!